osu.Game/Tests/Visual/ModTestScene.cs

```diff
@@ -72,6 +95,12 @@ protected class ModTestData
             /// </summary>
             public bool Autoplay = true;
 
+            /// <summary>
+            /// The frames to use for replay. <see cref="Autoplay"/> must be set to false.
+            /// </summary>
+            [CanBeNull]
+            public List<ReplayFrame> Frames;
```

Comment:
why `List<ReplayFrame>` instead of just a `Replay`?

Reply:
Its much more simpler to pass the frames itself since there's no need to modify anything under `Replay.ScoreInfo`.

Reply:
i suppose? would probably name this `ReplayFrames` rather than `Frames` myself, though